mobile/src/libs/components/sign-background/sign-background.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { type FC, type ReactNode } from 'react';
+import { Image, type ImageSourcePropType } from 'react-native';
+
+import bottomBubbleImg from '#assets/bottom-bubble.png';
+import backgroundImg from '#assets/sign-background.png';
+import topBubbleImg from '#assets/top-bubble.png';
+import { View } from '#libs/components/components';
+
+import { styles } from './styles';
+
+type Properties = {
+  children: ReactNode;
+};
+
+const SignBackground: FC<Properties> = ({ children }) => {
```

Comment:
It is better to use one approach and use React.FC (as an example), as was discussed in the [slack](https://bsa2023.slack.com/archives/C05P563E2MN/p1692877402951959)